harvest/create-harvesters.py

```diff
@@ -1,9 +1,15 @@
 import argparse
 import json
 import requests
+import time
 import yaml
+from datetime import datetime
+
+# FIXME
+# getting requests errors (remote closed) after large time.sleep
+# session = requests.Session()
+session = requests
```

Comment:
Makes sense. I'd remove the FIXME and rename session to requests. We don't need a session and stateless requests are the way to go.